__tests__/app.test.js

```diff
@@ -229,3 +229,52 @@ describe('7-post', () => {
     .expect(404)
   })
 })
+
+describe('8-patch', () => {
+  test('status 200 with positive number', () => {
+    const voteInc = {voteInc: 5}
+    return request(app)
+    .patch('/api/reviews/3')
+    .send(voteInc)
+    .expect(200)
+    .then(({body}) => {
+      expect(body.review).toEqual(
+        expect.objectContaining({
+          votes: 10,
+          category:expect.any(String),
+          created_at:expect.any(String),
+          review_id:expect.any(Number),
+          designer:expect.any(String),
+          owner:expect.any(String),
+          review_id: 3,
+          review_img_url:expect.any(String),
+          title:expect.any(String),
+          review_body:expect.any(String)
+        })
+      )
+
```

Comment:
I'd like to see an additional test in here which shows that decreasing the votes also works... i.e. if you send voteInc: -5

does that decrease the vote count by 5, for example?

Reply:
Jim told me this was a wasted test as it essentially just a repeat of the first test